src/backend/commands/createas.c

```diff
@@ -388,7 +388,7 @@ ExecCreateTableAs(CreateTableAsStmt *stmt, const char *queryString,
 							&& strcmp(aggname, "count") != 0
 							&& strcmp(aggname, "avg") != 0
 						)
-							elog(ERROR, "Aggrege function %s is not supported", aggname);
+							elog(ERROR, "aggrege function %s is not supported", aggname);
```

Comment:
This should be fixed to "aggregate" not "aggrege", as pointed out by Thomas.

> s/Aggrege/aggregate/

This is same in regression test fixes.